app/view_models/auctions_index_view_model.rb

```diff
@@ -1,29 +1,30 @@
-class AuctionsIndexViewModel < Struct.new(:current_user, :auctions_query)
-  def auctions
-    @auctions ||= auctions_query.map { |auction| AuctionViewModel.new(current_user, auction) }
+class AuctionsIndexViewModel
+  attr_reader :auctions, :current_user
+
+  def initialize(auctions:, current_user:)
+    @auctions = auctions.to_a
```

Comment:
so moving `to_a` into `public_index` breaks the unit tests for the query

convo at https://18f.slack.com/archives/micropurchase-dev/p1464120732004794

vote for tabling rather than falling down this rabbit hole atm